packages/platform/atoms/booker/BookerWebWrapper.tsx

```diff
@@ -48,7 +48,9 @@ export const BookerWebWrapper = (props: BookerWebWrapperAtomProps) => {
   useEffect(() => {
     // This event isn't processed by BookingPageTagManager because BookingPageTagManager hasn't loaded when it is fired. I think we should have a queue in fire method to handle this.
     sdkActionManager?.fire("navigatedToBooker", {});
-  }, []);
+    //This removes the queryparameters from the url as soon as it gets added on every cliks on layout or calender dates
+    history.replaceState(null, "", window.location.pathname);
+  });
```

Comment:
Don't think it's a correct way to remove query params, also it's not working locally.

Reply:
Thanks for the feedback @Praashh  what if we then push the state instead of replacing the state on initial load only, as replaceState removes all the state.